bench/src/main/scala/Performance.scala

```diff
@@ -16,7 +17,7 @@ object jsdom extends js.Object {
   def jsdom(innerHTML: js.UndefOr[String]): js.Any = js.native
 }
 
-object Performance {
+object Performance extends HandlerOps[IO] {
```

Comment:
Performance could be turned into `class Performance[F[_]: Sync] extends HandlerOps[F]`, and then we could test if there are performance differences between various implementations (IO, Task, SyncIO, etc) (see #321).

Reply:
This will be addressed as a separate PR, too many things change due to use of unsafeRunSync in bunches of places inside this object.